.patches/beginner/ex00.cairo.patch

```diff
@@ -1,5 +1,5 @@
---- exercises/beginner/ex00.cairo	2022-05-10 13:54:32.000000000 +0200
-+++ .solutions/ex00.cairo	2022-05-10 13:56:26.000000000 +0200
```

Comment:
I think we shouldn't commit all these changes to the patches. They just change the patch time and the filepath.